app/models/user.rb

```diff
@@ -7,4 +7,8 @@ class User < ApplicationRecord
 
   has_many :transactions, dependent: :destroy
   has_many :authorizations, dependent: :destroy
+
+  def user_balance
+    transactions.sum(:sum)
```

Comment:
@sinpai 
```ruby
sum(:sum)
```
looks strange 😄 

what do you think about renaming `sum` column to `amount`? for clear view